index.html

```diff
@@ -37,9 +43,235 @@ <h1>Donice Ubaru</h1>
       <button type="button">Start Collaboration</button>
     </div>
 
-    <img src="./images/geometry.svg" id="geometry-image" alt="geometry">
+    <img src="./images/geometry.svg" alt="geometry">
+  </section>
+
+  <section id="Works">
+    <h2>Projects</h2>
+
+    <div class="project-card-section">
+      <div class="project-card">
+        <img src="./images/project-image.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
```

Comment:
- KIndly fix the spacings in this button.

- Your design
<img width="106" alt="image" src="https://github.com/donice/Portfolio-Microverse/assets/41163571/0b1b1c49-b51c-4d5c-8b19-685d3f57b581">

- Figma design
<img width="92" alt="image" src="https://github.com/donice/Portfolio-Microverse/assets/41163571/7ef322cc-090d-4cf1-9f28-c16426ee22b5">